Tests/Tests/ViewSpec.swift

```diff
@@ -26,19 +26,19 @@ class ViewSpec: QuickSpec {
                 it("will save onDismiss action") {
                     waitUntil { done in
                         let onDismiss: (Bool) -> Void = { _ in done() }
-                        view.show(accessibilityCompatible: false, parentView: parentView, onDismiss: onDismiss)
```

Comment:
Why was accessibilityCompatible removed?

Reply:
Because it was never used. It's a leftover from refactoring some time ago. Now `accessibilityCompatible` flag is handled by Router, not by view classes.